sunpy/io/src/ana/_pyana.c

```diff
@@ -342,3 +349,4 @@ static PyObject * pyana_fzwrite(PyObject *self, PyObject *args) {
 	// If we didn't crash up to here, we're probably ok :P
 	return Py_BuildValue("i", 1);
 }
+#endif
```

Comment:
I believe the error is because this endif is now down here. `PyInit__pyana` is defined on line 105 and it's not being hit because this if statement is excluding all the Python wrapper code if `__USE_GNU` flag is not set on line 12.

Reply:
Yeah, I too think that was the case, the pyhton wrappers were too mixed up with __USE_GNU